lib/ansible/modules/network/nsupdate.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/python
+
+"""
+Ansible module to manage DNS records using dnspython
+(c) 2016, Marcin Skarbek <github@skarbek.name>
+(c) 2016, Andreas Olsson <andreas@arrakis.se>
+(c) 2017, Loic Blot <loic.blot@unix-experience.fr>
+
+This module was ported from https://github.com/mskarbek/ansible-nsupdate
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: nsupdate
+
+short_description: Setup OMAPI hosts.
```

Comment:
Short description - this should at least mention DNS.  Not sure how OMAPI is relevant here?